sv_pipeline/exome/load_data.py

```diff
@@ -106,16 +107,25 @@ def _parse_genes(genes):
         'field_name': GENES_FIELD,
         'format': _parse_genes,
     },
+}
+
+NEW_JOINT_CALL_SAMPLE_COLS = {
     PREV_IDENTICAL_COL: {'field_name': 'prev_call', 'format': bool},
     PREV_OVERLAP_COL: {'field_name': 'prev_overlap', 'format': bool},
     PREV_MISSING_COL: {'field_name': NEW_CALL_FIELD, 'format': lambda call: call not in {'NA', 'FALSE'}},
 }
+
+MERGING_CALL_SAMPLE_COLS = {
+    IS_LATEST: {'field_name': 'prev_call', 'format': lambda val: not BOOL_MAP[val.strip()]},
```

Comment:
this is incorrect, this should be `PREV_IDENTICAL_COL`. The column names are the same regardless of how we are doing the loading, its just how they are computed that is different

Reply:
The `PREV_IDENTICAL_COL` is the column in the input file. The output column `prev_call` doesn't change.

Reply:
I see. In that case you need to make constants for `prev_call` and `prev_overlap`, you shouldn't be copy-pasting strings